Tests/Unit/EmogrifierTest.php

```diff
@@ -956,10 +956,27 @@ public function emogrifyForInvalidCssSelectorThrowsException()
             '<html><style type="text/css">p{color:red;} <style data-x="1">html{cursor:text;}</style></html>'
         );
 
+        $this->subject->setDebug(true);
         $this->subject->emogrify();
     }
 
     /**
+     * @test
+     */
+    public function emogrifyIgnoreInvalidCssSelectors()
```

Comment:
I kept the test from pull request #361 "Handling invalid xPath expression warnings" and added the following line to test debug mode :
`$this->subject->setDebug(true);`

See lines 948 to 961

>